tests/Feature/Planner/GenerateDailyPlanJobTest.php

```
@@ -0,0 +1,42 @@
+    public function test_the_plan_date_is_parsed_in_the_user_timezone(): void
+    {
+        $user = User::factory()->create();
+        $user->company->settings()->set(
+            CompanySettingsEnum::LOCALIZATION_TIMEZONE->value,
+            'America/Sao_Paulo',
+        );
+
+        $generator = Mockery::mock(DailyPlanGenerator::class);
+        $generator->shouldReceive('generate')
+            ->once()
+            ->withArgs(function (Company $company, CarbonImmutable $date, DailyPlanModeEnum $mode): bool {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$company'. (undefined)

(UnusedFormalParameter)